Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'json'
 gem 'aws-sdk'
```

Comment:
Can we remove the mega gem, since we only need `aws-sdk-kms`?

Reply:
Good idea, I've removed that